en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,201 @@
+
+#### Slew Rates
+
+Slew rates limit how fast the signal that is sent to the motors is allowed to change:
+
+- [RA_MAX_ACCEL](../advanced_config/parameter_reference.md#RA_MAX_ACCEL): This is the limit of how fast the rover can accelerate (this is not based on acceleration measurements but on assumed linear relation between the throttle and the maximum rover speed).
+
+:::warning
+[RA_MAX_SPEED](../advanced_config/parameter_reference.md#RA_MAX_SPEED) needs to be set for this to work!
+:::
+
+- [RA_MAX_ACCEL](../advanced_config/parameter_reference.md#RA_MAX_ACCEL): This is the limit on the steering rate.
+
+### Mission Parameters
+
+These parameters only affect the mission flight mode.
+
+:::warning
+The parameters in [Tuning (basic)](#tuning-basic) have to be set to drive missions!
+:::
+
+#### Pure Pursuit Parameters
+
+The module uses a control algorithm called _Pure pursuit_ and it can be beneficial to understand how it works to properly tune (see [Pure pursuit algorithm](#pure-pursuit-algorithm)).
+The look ahead distance sets how aggressive the controller behaves and is defined as follows:
+
+$$
+l_d = v \cdot k
+$$
+
+It depends on the velocity $v$ of the rover and a tuning parameter $k$ that can be set with the parameter [RA_LOOKAHEAD_TUN](#RA_LOOKAHEAD_TUN).
+
+:::info
+A lower value of `RA_LOOKAHEAD_TUN` makes the controller more aggressive but can lead to oscillations!
+:::
+
+To deal with the edge case that the line segment is outside the look ahead radius around the rover there are 2 steps:
+
+1. The look ahead is scaled to the crosstrack error, which creates an intersection point for the rover to drive towards.
+2. Scaling the look ahead too much can have undesirable effects such as the rover driving a huge circle to return to the path, rather then taking a straight line. For this purpose the maximum look ahead radius can be capped with the parameter [RA_LOOKAHEAD_MAX](#RA_LOOKAHEAD_MAX), after which the rover will drive directly towards the current waypoint until it gets close enough to the line segment again.
+
+To summarize, the following parameters can be used to tune the controller:
+
+| Parameter                                                                                                   | Description                             | Unit |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------- | ---- |
+| <a id="RA_LOOKAHEAD_TUN"></a>[RA_LOOKAHEAD_TUN](../advanced_config/parameter_reference.md#RA_LOOKAHEAD_TUN) | Main tuning parameter                   | -    |
+| <a id="RA_LOOKAHEAD_MAX"></a>[RA_LOOKAHEAD_MAX](../advanced_config/parameter_reference.md#RA_LOOKAHEAD_MAX) | Maximum value for the look ahead radius | m    |
+| <a id="RA_LOOKAHEAD_MIN"></a>[RA_LOOKAHEAD_MIN](../advanced_config/parameter_reference.md#RA_LOOKAHEAD_MIN) | Minimum value for the look ahead radius | m    |
+
+#### Cornering Parameters
```

Comment:
Note, first letter of all words are capitalised. Generally "related" is unnecessary in the heading - they are "Cornering paramters".